static/self-check.html

```diff
@@ -17,7 +17,7 @@
 	</head>
 	
 	<body onload="javascript:initiate();">
-		<div class="toptoolbar">
+		<!--<div class="toptoolbar">-->

```

Comment:
You comment out the start of the div, but not the end of the div. Could be useful to run the html through a linter.